### DISCUSSION TITLE
Положительная и отрицательная

### DISCUSSION THREAD

**Discussion by ItzIsonn** [iBuseWinner/Reputation#17] [Ideas]
Предлагаю добавить возможность отключить такой вариант событий: игрок дал второму +1, а через секунду он же (первый) забрал у второго -1.
Тоже параметр типа такого: enable_one_vote: true (включить только один голос)

**Comment by iBuseWinner** [comment]
Будет реализовано в ближайшее свободное время.

**Comment by iBuseWinner** [comment]
Реализовано в версии 3.0.8-БЕТА.
